packages/cli/src/controllers/telemetry.controller.ts

```diff
@@ -0,0 +1,59 @@
+import { GlobalConfig } from '@n8n/config';
+import { AuthenticatedRequest } from '@n8n/db';
+import { Get, Post, RestController } from '@n8n/decorators';
+import { NextFunction, Response } from 'express';
+import { createProxyMiddleware, fixRequestBody } from 'http-proxy-middleware';
+
+@RestController('/telemetry')
+export class TelemetryController {
+	proxy;
+
+	constructor(private readonly globalConfig: GlobalConfig) {
+		this.proxy = createProxyMiddleware({
+			target: this.globalConfig.diagnostics.frontendConfig.split(';')[1],
+			changeOrigin: true,
+			pathRewrite: {
+				'^/proxy/': '/', // /proxy/v1/track -> /v1/track
+			},
+			on: {
+				proxyReq: (proxyReq, req) => {
+					proxyReq.removeHeader('cookie');
+					fixRequestBody(proxyReq, req);
+					return;
+				},
+			},
+		});
+	}
+
+	@Post('/proxy/:version/track', { skipAuth: true, rateLimit: true })
```

Comment:
With `rateLimit: true`, we use the default values for express-rate-limit. Isn't 5 requests per 1 minute too restrictive?

Reply:
ahh good catch! Not sure what would be the ideal here, but what do you think of 100 requests per minute?

Reply:
increased the limits for /track and /page calls